jenkins/jobs/parallel_e2e_features_test.pipeline

```diff
@@ -243,7 +243,8 @@ pipeline {
           environment {
             TEST_EXECUTER_VM_NAME = "${TEST_EXECUTER_VM_NAME}-live-iso"
             BUILD_TAG = "${env.BUILD_TAG}-live-iso"
-            GINKGO_FOCUS = "live-iso"
+            GINKGO_FOCUS = "features"
+            GINKGO_SKIP = "pivoting remediation"
```

Comment:
The `stage` says this is e2e live iso test. I'm not understanding this. Is live-iso part of features? And does it work now to use both focus and skip? :thinking:

Reply:
Thanks for spotting that ! the third stage should run feature tests that are not part of pivoting-based feature nor remediation features for instance live-iso and IPReuse that are feature tests but not part of pivoting-based and remediation.
I expect it to work with both first focus on all features and exclude skipped once so this stage should run all the features tests excluding pivoting based and remediation

Reply:
fixed

Reply:
Thanks for explaining, sounds good! The `BUILD_TAG` should also be changed though (line 245)